packages/labs/react/tsconfig.json

```diff
@@ -22,7 +22,8 @@
     "experimentalDecorators": true,
     "jsx": "react",
     "noImplicitOverride": true,
-    "types": ["mocha", "react", "react-dom"]
+    "skipLibCheck": true,
```

Comment:
Why was this needed? Is there an issue with the `react`, and `react-dom` types being checked as they've been removed from the types array?

Reply:
`react` and `react-dom` could be removed from `types` as we're not relying on them being available as UMD globals.
`skipLibCheck` was needed because some React internal typings were causing errors, possibly incompatible with our other tsconfig options.